sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventHubProducerAsyncClientIntegrationTest.java

```diff
@@ -247,4 +252,41 @@ void worksAfterReconnection() throws InterruptedException {
         TimeUnit.MINUTES.sleep(30);
         System.out.println("Complete.");
     }
+
+    @Disabled("Testing idle/long running operations.")
```

Comment:
Would this be better suited in our stress test client since this is disabled and wouldn't run in any of our live tests?

Reply:
Yes, it would for our new stress infrastructure. But I didn't want to manually have to fire up their giant infrastructure when I wanted to test a change. :( I'll create an issue to move these tests.